compose/config/config.py

```diff
@@ -292,7 +293,7 @@ def load(config_details):
     config_details = config_details._replace(config_files=processed_files)
 
     main_file = config_details.config_files[0]
-    volumes = load_mapping(config_details.config_files, 'get_volumes', 'Volume')
+    volumes = load_volumes(config_details.config_files)
     networks = load_mapping(config_details.config_files, 'get_networks', 'Network')
```

Comment:
I just noticed we have the same issue with network.driver_opts. Probably makes sense to fix it here as well?